chanjo_report/server/blueprints/report/utils.py

```diff
@@ -105,6 +105,7 @@ def keymetrics_rows(samples_ids, genes=None):
             func.avg(TranscriptStat.completeness_50).label('completeness_50'),
             func.avg(TranscriptStat.completeness_100).label('completeness_100'),
         )
+        .with_hint(TranscriptStat, "USE INDEX (_sample_transcript_uc)")
```

Comment:
💯 DuckDuckGo agrees - I used to work with Postgresql so this index hinting is Other Side magic to me. 😄

Reply:
> 💯 DuckDuckGo agrees 

 OK .. ! 😐

Reply:
Before merging let me just check real quick that the same index exists also on prod db, just to be sure!

Reply:
Yes! Also because stage must have been copied from prod at some point, but I wanted to be sure!

<img width="554" alt="image" src="https://github.com/Clinical-Genomics/chanjo-report/assets/28093618/76a3545d-9ebb-4f17-bda4-a14c9a7fbb77">